src/vmm/src/cpu_config/x86_64/cpuid/intel/normalize.rs

```diff
@@ -363,11 +370,11 @@ mod tests {
     )]
 
     use super::*;
-
     #[test]
     fn default_brand_string_test() {
+        const DEFAULT_BRAND_STRING_BASE: &[u8; 28] = b"Intel(R) Xeon(R) Processor @";
```

Comment:
I think it is worth making `DEFAULT_BRAND_STRING_BASE` a usual static constant, so there is not need to copy it for all tests.